cpu/stm32f0/periph/gpio.c

```diff
@@ -195,7 +202,7 @@ void isr_exti(void)
     for (size_t i = 0; i < EXTI_NUMOF; i++) {
```

Comment:
Not necessarily  you can use the EXTI also to only check for line changes,
or in combination with the WFE instruction. This is done by enabling EMR
and not IMR. If the event occurs on an masked our ISR and another interrupt
is triggered, the IRQ handler will call the EXTI handler (which could be
NULL) which didn't want IRQs. This since the PR will be set.
On 14 Feb 2016 13:06, "Hauke Petersen" <notifications@github.com> wrote:

> In cpu/stm32f0/periph/gpio.c
> <https://github.com/RIOT-OS/RIOT/pull/4698#discussion_r52841680>:
>
> > @@ -195,7 +202,7 @@ void isr_exti(void)
> >      for (size_t i = 0; i < EXTI_NUMOF; i++) {
>
> AFAIK this is not needed, because if you disable a line, the corresponding
> bit in the EXTI->PR is not set on any event...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/4698/files#r52841680>.
>